.github/actions/checkout-helm-versions/action.yml

```diff
@@ -0,0 +1,39 @@
+# This action supports helmVersions reporting in the TestRunner BigQuery schema. For example,
+# https://console.cloud.google.com/bigquery?cloudshell=false&project=terra-kernel-k8s&ws=!1m10!1m4!4m3!1sterra-kernel-k8s!2ssimple_stream_dataset!3ssimple_streamtable!1m4!4m3!1sterra-kernel-k8s!2ssimple_stream_dataset!3sSUMMARY_testRun
+name: 'checkout-helm-versions'
+description: 'Action to do a sparse checkout of environments and versions directories from https://github.com/broadinstitute/terra-helmfile/'
+author: 'ichengchang'
+inputs:
+  repository:
+    description: 'terra-helmfile repo'
+    required: false
+    default: 'broadinstitute/terra-helmfile'
+  ref:
+    description: 'The branch, tag or SHA to checkout'
+    required: false
+    default: 'master'
+  token:
+    description: 'Access token used to fetch the repository'
+    required: true
+  path:
+    description: 'Relative path under $GITHUB_WORKSPACE to place the checkout directories'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Sparse checkout https://github.com/broadinstitute/terra-helmfile/
+      id: sparse-checkout
+      run: |
+        mkdir -p ${{ inputs.path }}
+        cd ${{ inputs.path }}
+        git init
+        git remote add origin https://${{ inputs.token }}@github.com/${{ inputs.repository }}
+        git fetch --all --quiet
+        git config core.sparseCheckout true
+        if [ -f .git/info/sparse-checkout ]; then
+          rm .git/info/sparse-checkout
+        fi
+        echo "environments/" >> .git/info/sparse-checkout
```

Comment:
won't this fail if you've deleted the sparse-checkout file?

Reply:
The purpose is to create a fresh `sparse-checkout` file with only the two subdirectories `environments` and `versions` so git will only checkout those two directories from `terra-helmfile`. Tested and working as expected.